### PR TITLE
Update extended marker and romanisation metadata rules in `Ranking criteria`

### PR DESCRIPTION
**Pull request by Protastic101**
Making PRs for these two metadata RC proposals that have been in the making for a while now https://osu.ppy.sh/community/forums/topics/1750611?n=1 and https://osu.ppy.sh/community/forums/topics/1743222?n=1. Tldr, adding extended markers for unofficial, player made extensions of songs, and adding/clarifying romanisation for Nordic characters.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Ranking_Criteria/en.md [hunk 1]
```diff
@@ -120,7 +121,8 @@ Important to understand before using:
 - **When romanising an artist's name yourself and no official romanisation is available, it must be romanised in the order it is printed in the unicode field.**
 - **Loan words from other languages have to use the original words in their stead when attempting to romanise them.**
 - **When a song uses repeat words in the title or in the artist where one is in unicode, and the other as a basic romanisation, the romanised field must use the provided romanisation only and remove the duplicate word.**
-- **Umlauts must be romanised into two-letter equivalents: `ü` to `ue`, `ö` to `oe`, `ä` to `ae` and `ß` to `ss`.**
+- **Umlauts must be romanised into two-letter equivalents: `ü` to `ue`, `ö` to `oe`, `ä` to `ae` and `ß` to `ss`.** In Swedish and Finnish, `ö` and `ä` should instead be romanised to `o` and `a` respectively.
+- **Metadata containing Nordic letters must be romanised to the following: `æ` to `ae`, `ø` to `oe`, `å` to `aa`.** If it is Swedish, `å` should instead be romanised to `a`.
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/en.md:125]:
```suggestion
- **Umlauts must be romanised into two-letter equivalents: `ü` to `ue`, `ö` to `oe`, `ä` to `ae`, and `ß` to `ss`.** In Swedish and Finnish, `ö` and `ä` should instead be romanised to `o` and `a` respectively.
- **Metadata containing Nordic letters must be romanised to the following: `æ` to `ae`, `ø` to `oe`, and `å` to `aa`.** If it is Swedish, `å` should instead be romanised to `a`.
```

**Review by Hiviexd — APPROVED** [review]

**Comment by aceticke** [comment]
i dont think extended marker should be pushed just yet as there are still some points i want to bring up

**Comment by Protastic101** [comment]
> i dont think extended marker should be pushed just yet as there are still some points i want to bring up

Responded [here](https://osu.ppy.sh/community/forums/topics/1750611?n=16) but edit has been commonly used for unofficial changes to songs done by other players for a variety of reasons, while Ver. is often more synonymous with official releases and versions. I think it will cause more confusion to make it (Ver.) instead of (Edit) as you can see from the discussion in Oko's proposal.

**Comment by Hydria** [comment]
Point addressed in thread as mentioned, making it "Extended Ver." would just add _more_ confusion to official vs unofficial edits to the song.

**Comment by osuAntalf** [comment]
I think from what was discussed in the threads (take into account Oko's one as well) plus an approval from the community and NAT as well I think this is more than ready to be pushed.

**Comment by Protastic101** [comment]
Small update: We're gonna wait for Oko to finish finalizing and revising [his proposal](https://osu.ppy.sh/community/forums/topics/1762241?n=1) before pushing these proposals. We'll also include his finalized proposal into this PR for convenience.

**Comment by Walavouchey** [comment]
in case you feel like that discussion will take a while, splitting these changes and merging the romanisation part first is also an option

**Review by peppy — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]
was [told on discord](https://discord.com/channels/188630481301012481/218677502141399041/1115714377207795762) that this one's good to go as is without waiting for further changes, and there have been no updates on that anyway

**Pull request merged**